core/src/main/java/com/scalar/db/api/Admin.java

```diff
@@ -388,4 +388,13 @@ void addNewColumnToTable(String namespace, String table, String columnName, Data
    * @throws ExecutionException if the operation failed
    */
   Set<String> getNamespaceNames() throws ExecutionException;
+
+  /**
+   * Upgrades the Scalar DB environment to support the latest version of the Scalar DB API.
```

Comment:
Can you elaborate on what is upgrade and why we need this?
I just don't know about it, but it helps me to understand and review it properly.

Reply:
Thank you for your question. Sorry, I had prepared inlined comments but failed to publish them.

The motivation for adding this is that we wanted that users who update their existing Scalar DB application to the 4.0.0 version (planned release version for the namespace management feature) will be able to have `admin.getNamespaceNames()` returns namespaces created before and after the 4.0.0.
This means we needed to find a way for users to import their existing namespaces into the metadata namespaces table so that they can be managed via the admin interface.
Additionally, we may develop another feature in the future that requires as well to update the Scalar DB environment as the namespace management does. Therefore, we decided to keep a generic name for the `upgrade()` so that this method can be used for future features.

Reply:
Got it.Thank you!